index.bs

```diff
@@ -128,7 +140,25 @@ This callback will be invoked when there are changes to <i>target</i>'s intersec
     </pre>
 
     <div dfn-type="method" dfn-for="IntersectionObserver">
-        : <dfn>observe()</dfn>
+        : <dfn>Constructor(callback, options)</dfn>
```

Comment:
There's no established convention I know of for this sort of thing but I personally like the heading to be `new IntersectionObserver(callback, options)`.